pixeltable/share/packager.py

```diff
@@ -65,16 +66,16 @@ def package(self) -> Path:
         Export the table to a tarball containing Parquet tables and media files.
         """
         assert not self.tmp_dir.exists()  # Packaging can only be done once per TablePackager instance
-        _logger.info(f"Packaging table '{self.table._path}' and its ancestors in: {self.tmp_dir}")
+        _logger.info(f"Packaging table '{self.table._path()}' and its ancestors in: {self.tmp_dir}")
         self.tmp_dir.mkdir()
         with open(self.tmp_dir / 'metadata.json', 'w', encoding='utf8') as fp:
             json.dump(self.md, fp)
-        self.iceberg_catalog = sqlite_catalog(self.tmp_dir / 'warehouse')
+        self.tables_dir = self.tmp_dir / 'tables'
+        self.tables_dir.mkdir()
         with catalog.Catalog.get().begin_xact(for_write=False):
-            ancestors = (self.table, *self.table._base_tables)
-            for t in ancestors:
-                _logger.info(f"Exporting table '{t._path}'.")
-                self.__export_table(t)
+            for tv in self.table._tbl_version_path.get_tbl_versions():
+                _logger.info(f"Exporting table '{tv.get().name}:{tv.get().version}'.")
```

Comment:
Note: There's a `TableVersion.versioned_name` now, so you can do

```
'Exporting table {tv.get().versioned_name!r}.'
```

Reply:
Done.